smoke/snat44_test.sh

```
@@ -0,0 +1,50 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2024 Robin Jarry
+
+. $(dirname $0)/_init.sh
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)